x/tokenomics/keeper/token_logic_modules.go

```diff
@@ -20,6 +20,7 @@ import (
 	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 	suppliertypes "github.com/pokt-network/poktroll/x/supplier/types"
+	"github.com/pokt-network/poktroll/x/tokenomics"
```

Comment:
Does your IDE not auto sort imports on save?

Reply:
Just checked, it actually sorted it there.